templates/index.html

```diff
@@ -1,83 +1,89 @@
-  <meta name="twitter:description" content="{{ config.description }}">
-  <meta name="twitter:image" content="/images/logo/limeleaf-logo.jpg">
-  <link rel="me" href="https://social.coop/@applycoop" />
-  <link rel="me" href="https://social.coop/@limeleaf" />
+<title>{{ config.title }}</title>
+<meta property="og:title" content="{{ config.title }}" />
+<meta property="og:description" content="{{ config.description }}" />
+<meta property="og:url" content="https://limeleaf.coop" />
+<meta property="og:image" content="/images/logo/limeleaf-logo.jpg" />
+<meta property="og:image:width" content="1200" />
+<meta property="og:image:height" content="630" />
+<meta property="og:type" content="website" />
+<meta name="description" content="{{ config.description }}" />
+<meta name="twitter:card" content="summary_large_image">
+<meta property="twitter:domain" content="limeleaf.net">
+<meta property="twitter:url" content="https://limeleaf.coop">
+<meta name="twitter:title" content="{{ config.title }}">
+<meta name="twitter:description" content="{{ config.description }}">
+<meta name="twitter:image" content="/images/logo/limeleaf-logo.jpg">
+<link rel="me" href="https://social.coop/@applycoop" />
+<link rel="me" href="https://social.coop/@limeleaf" />
 {% endblock meta_content %}
 
 {% block content %}
 <main class="container">
-  <section class="hero-section">
-      <div class="row">
-          <div class="col-sm">
-              <h1 class="home">Your Rust and Go Development Experts</h1>
-              <p>We are a worker-owned cooperative that helps teams build fast, reliable, maintainable software using modern programming languages.</p>
-          </div>
-          <div id="hero-image" class="col-sm-5">
-              <img src="/images/services/backend-developer.svg"/>
-          </div>
-      </div>
-  </section>
-  <section class="home services-section">
-      <div class="col-sm">
-      <div class="row">
-        <div class="col-sm services consulting">
-          <h2 class="service-title">Consulting</h2>
-          <p class="service-description">Switching to Rust or Go? Need help with a complex project? From code reviews to infrastructure planning, we'll help you ship better software faster.</p>
-          <p><a href="/services/consulting/" class="link">Learn more</a></p>
+    <section class="hero-section">
+        <div class="row">
+            <div class="col-sm">
+                <h1 class="home">Your Fractional CTO for Complex Software Projects</h1>
+                <p>We are a worker-owned cooperative that helps teams build fast, reliable, maintainable software using modern programming languages and cutting-edge protocols like ATProto.</p>
+                <div class="hero-cta">
+                    <a href="/contact/" class="btn btn-primary">Start Your Project</a> | 
+                    <a href="/services/" class="btn btn-secondary">Our Services</a>
+                </div>
+            </div>
+            <div id="hero-image" class="col-sm-5">
```

Comment:
We should kill this stock art all together. This seems to be the only one on the site now. If we do that then the header text can go full width and not wrap.